website/sidebars.json

```diff
@@ -1,194 +1,173 @@
 {
-  "test1": [
-    "test1-section/prerequisites"
+  "overview": [
+    "overview-section/overview",
+    "overview-section/comparison",
+    "overview-section/differences"
   ],
 
-  "test2": [
-    {
-      "type": "category",
-      "label": "Issue verifiable documents with DID",
-      "items": [
-        "test2-section/intro",
-        "test2-section/create",
-        "test2-section/document-store-or-ocsp",
-        "test2-section/dns",
-        "test2-section/raw-document-did",
-        "test2-section/wrap-document-did",
-        "test2-section/sign-document",
-        "test2-section/additional",
-
-       [
+  "prereq": [
+    "prereq-section/prerequisites"
+  ],
+
+  "did": [
+
+        "did-section/overview-did",
+        "did-section/create",
+        "did-section/document-store-or-ocsp",
+        "did-section/dns",
+        "did-section/raw-document-did",
+        "did-section/wrap-document-did",
+        "did-section/sign-document",
+        "did-section/additional-did",
+       
           {
             "type": "category",
             "label": "Revoke documents issued using DID",
             "items": [
-              "test2-section/revoke-document-did/revoke-using-ocsp",
-              "test2-section/revoke-document-did/revoke-using-document-store"
+              "did-section/revoke-document-did/revoke-using-ocsp",
+              "did-section/revoke-document-did/revoke-using-document-store"
                 ]
               }
-            ]
-         ]
-      }
+
   ],
 
-  "test3": [
-    "test3-section/tutorial"
+  "ethereum": [
+
+        "ethereum-section/overview-eth",
+        "ethereum-section/wallet",
+        "ethereum-section/document-store",
+        "ethereum-section/dns-proof",
+        "ethereum-section/raw-document-eth",
+        "ethereum-section/wrap-document-eth",
+        "ethereum-section/issue-document",
+        "ethereum-section/additional-eth",
+
+          {
+            "type": "category",
+            "label": "Revoke documents issued using Ethereum",
+            "items": [
```

Comment:
here as well would be good to have it expanded by default

Reply:
OK